contracts/staking/StakeManager.sol

```diff
@@ -289,24 +281,21 @@ contract StakeManager is Validator, IStakeManager, RootChainable, Lockable {
 
   function withdrawRewards(uint256 validatorId) public onlyStaker(validatorId) {
     uint256 amount = validators[validatorId].reward;
-    address _contract = validators[validatorId].contractAddress;
-    if (_contract != address(0x0)) {
-      amount = amount.add(ValidatorContract(_contract).withdrawRewardsValidator());
-    }
     validators[validatorId].reward = 0;
     require(token.transfer(msg.sender, amount), "Insufficent rewards");
   }
 
-  function delegationTransfer(uint256 amount, address delegator) external {
+  function delegationTransfer(uint256 amount, address delegator) external returns (bool) {
```

Comment:
function name is confusing. pls change it to something like 'delgatorRewardWithdraw' or 'delegatorWithdrawal' etc

Reply:
Thanks fixed.